_search-plugins/sql/functions.md

```diff
@@ -33,20 +33,24 @@ ln | `ln(number T) -> double` | `SELECT ln(10) FROM my-index LIMIT 1`
 log | `log(number T) -> double` or `log(number T, number) -> double` | `SELECT log(10) FROM my-index LIMIT 1`
```

Comment:
This one too

Reply:
I believe that this one doesn't need to be updated as calling a `log(number T)` defaults the log to be base e

Reply:
Does `log(number T, number)` need to be `log(number T, number T)`?

Reply:
The `log` method can either have one or two parameters, if it has one it defaults to the natural log ln, and if you put in two parameters then one of the parameters is set to be the base